railties/test/generators/actions_test.rb

```diff
@@ -720,7 +720,11 @@ def test_log_with_status_with_quiet
 
   private
     def action(...)
-      capture(:stdout) { generator.send(...) }
+      if ENV["RAILS_LOG_TO_STDOUT"] == "true"
```

Comment:
Do we usually use `== "true"` or do we want to just assert this is not nil? I've seen people using env vars like `RAILS_LOG_TO_STDOUT=1 some_command`

```suggestion
      if !ENV["RAILS_LOG_TO_STDOUT"].nil?
```
Or, if we wanna go the Active Support route:

```suggestion
      if ENV["RAILS_LOG_TO_STDOUT"].present?
```

Reply:
In #49448 [we look for `"true"`](https://github.com/rails/rails/pull/49448/files#diff-be6904ff2570e3e8911209d2322c073d64f6d428f9cfecfde3bc8b8a4d93b4b6R71), so this commit is consistent with that one.

My original thinking was I didn't want something like `RAILS_LOG_TO_STDOUT=FALSE ./bin/test test/generators/actions_test.rb` to work.